tensorflow/python/keras/layers/recurrent_v2.py

```diff
@@ -1472,7 +1474,8 @@ def lstm_with_backend_selection(inputs, init_h, init_c, kernel,
     sequence_lengths: The lengths of all sequences coming from a variable length
       input, such as ragged tensors. If the input has a fixed timestep size,
       this should be None.
-    zero_output_for_mask: Boolean, whether to output zero for masked timestep.
+    
+    put_for_mask: Boolean, whether to output zero for masked timestep.
```

Comment:
Why is this docstring being changed.

Reply:
that was be accident, change has been dropped